### DISCUSSION TITLE
install  lava-decolle  (DECOLLE )  plugin

### DISCUSSION THREAD

**Discussion by bahgatayasi** [lava-nc/lava-dl#208] [Q&A]
Dear all,

I hope this email finds you well. I am currently facing difficulties installing the DECOLLE plugins in Lava-DL and have encountered several issues despite following the installation methods mentioned in the respective GitHub repositories.

I have made multiple attempts to install the Lava-DL library by referring to the instructions provided at https://github.com/lava-nc/lava-dl/. Similarly, I have also tried to install DECOLLE using the instructions at https://github.com/kclip/lava-decolle. Unfortunately, my efforts have been in vain as I have been unable to locate DECOLLE after installation. I have tried importing modules such as "DECOLLELoss" from the loss package and "DECOLLEAssistant" from the utils package, but I received error messages stating that the modules were not found.

I would greatly appreciate it if any of you could provide me with a clear tutorial or guide to successfully install the DECOLLE plugin. I am using the Anaconda GUI, Jupyter Notebook, and Python 3.9 included in Anaconda.

Thank you very much in advance for your kind assistance.

Best regards,

**Comment by bamsumit** [comment]
@bahgatayasi 
The correct way to import is
```python
from lava.lib.dl.decolle.loss import DECOLLELoss
from lava.lib.dl.decolle.utils.assistant import DECOLLEAssistant
```
It looks like lava-decolle examples do not to it. Please create an issue in the repo itself.
